src/hexkit/providers/mongodb/provider.py

```diff
@@ -53,20 +54,20 @@
 __all__ = [
     "MongoDbConfig",
     "MongoDbDaoFactory",
-    "translate_timeout_error",
+    "translate_pymongo_errors",
     "MongoDbDao",
 ]
 
 
 @contextmanager
-def translate_timeout_error():
+def translate_pymongo_errors():
     """Catch PyMongoError and re-raise it as DbTimeoutError if it is a timeout error."""
```

Comment:
```suggestion
    """Catch PyMongoError and re-raise it as appropriate DaoError subclass."""
```

Reply:
Fixed in [5daea3c](https://github.com/ghga-de/hexkit/pull/133/commits/5daea3c63eba2fda08ed7925a13e10ffa9883550)